### ISSUE TITLE
Error in Admin when categories array is emtpy

### ISSUE THREAD

**Issue by pmoreno-rodriguez** [OleVik/grav-theme-project-space#22]
Hello.
I have checked some of your code in another theme, specifically this code:

```
public static function categories()
     {
         $options = Grav::instance()['config']->get('themes.telephasic.categories');
         $categories = array();
         foreach ($options as $category) {
         $categories[$category] = $category;
         }
         return $categories;
     }
```

The thing is that if the theme option intended to insert the categories (declared in the blueprints.yaml file) is empty, the above code throws the following error:

```
An exception has been thrown during the rendering of a template ("foreach() argument must be of type array|object, null given").
```
I have made a simple modification that seems to solve this problem, as follows:

```
public static function categories()
     {
         $options = Grav::instance()['config']->get('themes.telephasic.categories');
         $categories = array();
         if (!empty($options)) {
            foreach ($options as $category) {
            $categories[$category] = $category;
            }
          }
         return $categories;
     }
```
This way I get the categories select field, even though it appears empty, to not show the previous error.

**Comment by OleVik** [comment]
That's an elegant improvement that should have been in place already, would you care to perform the change [directly to the code in GitHub](https://github.com/OleVik/grav-theme-project-space/blob/v2.0.4/project-space.php#L60), so it creates a pull request for merging?

**Comment by pmoreno-rodriguez** [comment]
[Pull Request 23](https://github.com/OleVik/grav-theme-project-space/pull/23) created

**Comment by OleVik** [comment]
Published as [v2.0.5](https://github.com/OleVik/grav-theme-project-space/releases/tag/v2.0.5), thanks!
